foodsaving/groups/models.py

```diff
@@ -96,6 +96,7 @@ class GroupMembership(BaseModel):
     group = models.ForeignKey(Group, on_delete=models.CASCADE)
     user = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
     roles = ArrayField(TextField(), default=list)
+    last_seen = DateTimeField(default=timezone.now)
```

Comment:
I would suggest to name this `lastseen_at` so it  is consistent with `ChannelSubscription.lastseen_at` field (and other `*_at` fields containing DateTimes)